include/flatbuffers/util.h

```diff
@@ -60,6 +60,21 @@ template<> inline std::string NumToString<signed char>(signed char t) {
 template<> inline std::string NumToString<unsigned char>(unsigned char t) {
   return NumToString(static_cast<int>(t));
 }
+#if defined(FLATBUFFERS_CPP98_STL)
+template <>
+inline std::string NumToString<long long>(long long t) {
+  char buf[21]; // (log((1 << 63) - 1) / log(10)) + 2
```

Comment:
Sorry, I was under the assumption the compiler would tolerate `log` in regular const expressions, that indeed doesn't make sense.